pages/osx/split.md

```diff
@@ -8,7 +8,7 @@
 
 - Split a file by a regular expression. The matching line will be the first line of the next output file:
 
-`split -p {{regular expression}} {{filename}}`
+`split -p {{cat|[d|h]og}} {{filename}}`
```

Comment:
@agnivade should `{{filename}}` be like `{{animals.txt}}`?

Reply:
Should `[d|h]` be `(d|h)`?

Reply:
good catch!

Reply:
so my test file was:
```
bat
cat
gnat
dog
hog
bog
dhog
```
and this worked for splitting only on cat, dog, and hog (and not splitting on dhog)